src/languages/singlestoredb/singlestoredb.formatter.ts

```diff
@@ -230,7 +233,7 @@ const reservedPhrases = ['ON DELETE', 'ON UPDATE', 'CHARACTER SET'];
 
 // https://docs.singlestore.com/managed-service/en/reference/sql-reference/comparison-operators-and-functions.html
 export default class SingleStoreDbFormatter extends Formatter {
-  static operators = ['~', '<=>', '<<', '>>', '&&', '||'];
+  static operators = ['~', ':=', '<=>', '<<', '>>', '&&', '||'];
```

Comment:
Could you reference a documentation for the `:=` operator, or provide an example of how it's used in SingleStoreDB.

Reply:
Found the answer. It's used in `SET @foo := 123;` syntax.